.circleci/test_runner.py

```diff
@@ -73,7 +77,17 @@ def get_envs() -> dict:
     test_runner = os.getenv('TEST_RUNNER_SERVICE')
     if test_runner is None:
         logging.critical('Test runner service ENV missing')
-        sys.exit(1)  # Return a non-zero exit code so CircleCI fails
+        raise Exception('Test runner service ENV missing')
+
+    envs['TEST_RUNNER_SLACK_WEBHOOK_01'] = os.getenv(
```

Comment:
What's Webhook 01 and Webhook 02?

Reply:
Its for #dev-plasma and #plasma-notifications I could make that into csv value if you prefer